mantaray/backend.py

```diff
@@ -199,6 +201,10 @@ def get_events(self) -> list[IrcEvent]:
         self._events.clear()
         return result
 
+    def get_nickmask(self) -> str:
+        assert self.nickmask is not None
```

Comment:
If the nickmask isn't available, it shouldn't be an error, so that hircd is still usable. Any code using the nickmask must be prepared to handle it being missing.

```suggestion
    def get_nickmask(self) -> str | None:
        if self.nickmask is None:
            return None
```